src/common/viz/FigureExtractor.js

```diff
@@ -180,6 +192,13 @@ define(['./Utils'], function (Utils) {
         }
     };
 
+    FigureExtractor.prototype.getMetadataChildrenIds = function (node) {
+        const metadataBaseNodePath = Object.keys(this._metaNodesMap)
+            .filter(id => this._metaNodesMap[id] === BASE_METADATA_TYPE).pop();
```

Comment:
Why not `.find`?

Reply:
My bad. I somehow always forget  [`Array.proptotype.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find).